collections/_learn/overview.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Overview"
+slug: "overview"
+layout: education
+hidden: false
+---
+
+# **Lesson 1: Overview**
+
+## Extending the EMR: A Developer’s Guide to the Canvas SDK
+
+### Introduction
+
+Welcome to Canvas! The Canvas platform is a fully-featured, certified Electronic Medical Record (EMR) that provides robust tools for customization and extension. For developers looking to integrate with or build custom logic within the EMR, the platform offers two main pathways: the **FHIR API** and the **Canvas SDK**.
+
+While the FHIR API offers a standards-based **REST API** with 37 resources (21 of which are writable) for interoperability, the Canvas SDK enables developers to write custom **Python programs (plugins)** that execute synchronously within the EMR server container.
+
+If your goal is to deeply customize the EMR’s behavior, implement clinical decision support, enforce business rules, or automate charting, the SDK is your primary tool.
+
+In this overview, we will explore the three core concepts developers need to master to build effective plugins: **Events**, **Data**, and **Effects**, and we'll look at the specific **Handler** types used to implement your logic.
+
+### The Plugin Runner: Your Custom Code in the EMR
+
+The core of the SDK is the **plugin runner**, an open-source gRPC server that executes your Python code. Your logic is structured as a set of one or more **handlers**—classes that perform a single responsibility and listen for declared events.
```

Comment:
it might be nice to use the term `effects` in here. something like `Your logic is structured as a set of one or more **handlers**-classes that listen for declared **events** and return one or more **effects**`